src/plugin.js

```diff
@@ -1,9 +1,39 @@
+/**
+* @overview ChatEngine Open Graph unfurls URLs sent in chat messages and transforms them into more interactive pieces of media. More info [here](https://www.pubnub.com/blog/open-graph-plugin-chatengine-unfurl-urls-chat-apps/).
```

Comment:
There's a tag called "@see" where you can add links like this. http://usejsdoc.org/tags-see.html